### PR TITLE
React capstone project - Metrics Web App

### PR DESCRIPTION
**Pull request by itsmraga-hub**
# 💯 Achievements

- [Link to loom video](https://www.loom.com/share/05944216de3b44efb0a244bfab493c32) explaining project features.
- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
```
Comment (Deepakdanger) [inline README.md]: - [x] Kindly change the title name to something appropriate.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Open your terminal and navigate to desired folder.
+
+- Clone the project from the repo `git clone https://github.com/itsmraga-hub/metrics-webapp.git`
+
+- Change directory to Math Mmagicians folder `cd metrics-webapp`.
```
Comment (Deepakdanger) [inline README.md:32]: - [x] `Math Mmagicians` keyword is used in readme , kindly name it properly.

File: src/components/Case.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable camelcase */
```
Comment (Deepakdanger) [inline src/components/Case.js]: - [x] Kindly remove it & solve the linter manually.

File: src/components/Menu.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-return-assign */
```
Comment (Deepakdanger) [inline src/components/Menu.js]: - [x] Kindly remove it & solve the linter manually.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @itsmraga-hub , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @itsmraga-hub, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights 💪 
- no linter errors
- project requirements are met
- good PR description
- good README


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by itsmraga-hub** [comment]
Thanks @BertrandConxy.

**Comment by BertrandConxy** [comment]
You are welcome @itsmraga-hub

**Pull request merged**